server/bundler/loader.js

```diff
@@ -1,76 +1,37 @@
-var config = require( 'config' ),
+const config = require( 'config' ),
+	fs = require( 'fs' ),
 	utils = require( './utils' );
 
 function getSectionsModule( sections ) {
-	var dependencies,
-		loadSection = '',
-		sectionLoaders = '';
+	let templateName,
+		sectionLoaders;
 
 	if ( config.isEnabled( 'code-splitting' ) ) {
-		dependencies = [
-			"var config = require( 'config' ),",
-			"\tpage = require( 'page' ),",
-			"\tReact = require( 'react' ),",
-			"\tactivateNextLayoutFocus = require( 'state/ui/layout-focus/actions' ).activateNextLayoutFocus,",
-			"\tLoadingError = require( 'layout/error' ),",
-			"\tcontroller = require( 'controller' ),",
-			"\trestoreLastSession = require( 'lib/restore-last-path' ).restoreLastSession,",
-			"\tpreloadHub = require( 'sections-preload' ).hub;",
-			'\n',
-			'var _loadedSections = {};\n'
-		].join( '\n' );
-
-		sections.forEach( function( section ) {
-			loadSection += singleEnsure( section.name );
-			section.paths.forEach( function( path ) {
-				sectionLoaders += splitTemplate( path, section );
-			} );
-		} );
-
-		return [
-			dependencies,
-			'function preload( section ) {',
-			'	switch ( section ) {',
-			'	' + loadSection,
-			'	}',
-			'}',
-			'\n',
-			"preloadHub.on( 'preload', preload );",
-			'\n',
-			'module.exports = {',
-			'	get: function() {',
-			'		return ' + JSON.stringify( sections ) + ';',
-			'	},',
-			'	load: function() {',
-			'		' + sectionLoaders,
-			'	}',
-			'};'
-		].join( '\n' );
+		templateName = 'loader-code-splitting.js.template';
+		sectionLoaders = getSectionLoadersCodeSplitting( sections );
+	} else {
+		templateName = 'loader.js.template';
+		sectionLoaders = getSectionLoaders( sections );
 	}
 
-	dependencies = [
-		"var config = require( 'config' ),",
-		"\tpage = require( 'page' ),",
-		"\tcontroller = require( 'controller' );\n"
-	].join( '\n' );
+	stringSections = JSON.stringify( sections );
 
-	sectionLoaders = getRequires( sections );
+	// Load the template and convert to a template literal using eval
+	return eval( '`' + fs.readFileSync( __dirname + '/' + templateName, 'utf8' ).toString() + '`' );
```

Comment:
I'm kinda debating here whether it's worth having an `eval` here over just doing string replace, since we only have 2 variables.

@Tug @stephanethomas what do you think?